teachers_digital_platform/jinja2/teachers_digital_platform/activity_index_page.html

```diff
@@ -40,26 +40,16 @@
                         {{ render_block.render(block, loop.index) }}
                     {% endif %}
                 {%- endfor %}
-                <div class="m-notification m-notification__visible m-notification__warning u-mt30">
-                    <span class="a-icon">{{ svg_icon('warning-round') }}</span>
-                    <div class="m-notification_content">
-                        <div class="m-notification_message">
-                            <h2 class="h4">Currently, there are only activities for high school.</h2>
-                            <p>Activities for elementary and middle school are being developed and will be available at a later time.</p>
-                        </div>
-                    </div>
-                </div>
             </div>
             <div class="m-search-hero_image-wrapper">
-                <img class="m-search-hero_image"
-                    srcset="{{ static('tdp/img/search-landing-hero.jpg') }},
-                        {{ static('tdp/img/search-landing-hero-2x.jpg') }} 2x"
-                    src="{{ static('tdp/img/search-landing-hero-2x.jpg') }}"
-                    width="498"
-                    alt="">
+                {% for block in page.header_sidebar -%}
+                    {% if block.block_type == 'image' %}
```

Comment:
Why is this if statement necessary? Aren't all blocks in the header_sidebar of type image?

Reply:
You're correct, it's not necessary. I'll remove it now.